Dockerfile

```diff
@@ -20,7 +20,6 @@ COPY . /usr/src/app
 #attempt_download('weights/yolov5m.pt'); \
 #attempt_download('weights/yolov5l.pt')"
 
-
```

Comment:
This line is still deleted. Just add it back in to remove the diff completely :)

Reply:
Fixed c6cd5fd716e138a1686ae322a8894e5c135dca8b

Reply:
Fixed c6cd5fd716e138a1686ae322a8894e5c135dca8b